db/migrate/20160805190034_create_join_table_fruits_orders.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableFruitsOrders < ActiveRecord::Migration[5.0]
+  def change
+    create_join_table :fruits, :orders do |t|
+      t.integer :quantity, default: 0
```

Comment:
This column keeps track of how much of one type of fruit is in an individual's order.